django_dynamic_fixture/django_helper.py

```diff
@@ -7,11 +7,15 @@
 
 import django
 from django.apps import apps
-from django.db import models
+from django.db import models  # noqa
 from django.db.models import *
-from django.db.models.fields import NOT_PROVIDED, AutoField, FieldDoesNotExist
+from django.db.models.fields import NOT_PROVIDED, AutoField
 from django.db.models.base import ModelBase
 from django.db.models.query import QuerySet
+try:
```

Comment:
I would suggest to remove the try/except block and only import from `django.core.exceptions`, as the exception was moved there 6 years ago. Only the legacy reference in `django.db.models.fields` was widely used and is removed with the 3.1 release.